<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltroInformacion</class>
 <widget class="QDialog" name="FiltroInformacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtro Datos</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="TrabajoPerifericos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>751</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>F_Repeparacion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Agente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cant</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NS</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>F_Fabricacion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Version</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Falla</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bonificacion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Garantia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Obsevaciones</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lFechaInicio">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>12</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lFechaFin">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>41</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Inicio </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Final</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bFechaInicio">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>11</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bFechaFin">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReportesFecha">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reporte por Fecha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReportesExcel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reportes Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ingresos">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reportes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reportes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IngresoFecha">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingreso por Fecha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IngresoExcel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingreso Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AjusteFechas">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajuste Fechas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FiltroInformacion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FiltroInformacion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
